com.brave.Brave.yaml

```diff
@@ -0,0 +1,119 @@
+app-id: com.brave.Brave
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: brave
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--env=GTK_PATH=/app/lib/gtkmodules'
+  - '--socket=cups'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--own-name=org.mpris.MediaPlayer2.chromium.*'
+  - '--system-talk-name=org.freedesktop.Avahi'
+  - '--system-talk-name=org.freedesktop.GeoClue2'
+  - '--system-talk-name=org.freedesktop.UPower'
+  - '--filesystem=host-etc:ro'
+  - '--filesystem=xdg-desktop'
+  - '--filesystem=xdg-download'
+  - '--filesystem=~/.local/share/applications/'
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: libcups
+    make-args: [libs]
+    no-make-install: true
+    post-install:
+      - make install-headers install-libs
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: https://github.com/apple/cups/releases/download/v2.3.3/cups-2.3.3-source.tar.gz
+        sha256: 261fd948bce8647b6d5cb2a1784f0c24cc52b5c4e827b71d726020bcc502f3ee
+
+  - name: gtk-cups-backend
+    buildsystem: meson
+    make-args: [modules/printbackends/libprintbackend-cups.so]
+    no-make-install: true
+    post-install:
+      - install -Dm 755 modules/printbackends/libprintbackend-cups.so -t /app/lib/gtkmodules/3.0.0/printbackends/
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/gtk.git
+        tag: 3.24.23
+
+  - name: gtk-settings
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 644 gtk-settings.ini /app/etc/xdg/gtk-3.0/settings.ini'
+    sources:
+      - type: file
+        path: gtk-settings.ini
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        branch: master
+        commit: 61be988e5de65aaec4d562db3d9c9c74357f6b01
+        # tag: v2020.10beta.3
+      # - type: dir
+      #   path: ../../code/zypak
+
+  - name: brave
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 644 brave_lion.svg /app/share/icons/hicolor/scalable/apps/com.brave.Brave.svg'
+      - 'install -Dm 755 brave.sh /app/bin/brave'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 755 update-desktop-database /app/bin/update-desktop-database'
```

Comment:
I wonder why would you need these for?

Reply:
you could add web apps that way. These "installable" web pages like Twitter, Element/Riot.im, etc.

Reply:
If you have the flatpak you should be able to install web pages to desktop/application menu.

but you will need the latest build: `flatpak install --user https://dl.flathub.org/build-repo/27502/com.brave.Brave.flatpakref
`

Reply:
> you could add web apps that way. These "installable" web pages like Twitter, Element/Riot.im, etc.

No, you cannot. The application has no access to host directory with desktop launchers. Same about `xdg-*` scripts.

Reply:
Removed